exec/src/coqstr.ml

```diff
@@ -44,32 +44,34 @@ let char_to_coqascii char =
   in
   mkApp (cAscii, of_list @@ List.rev (int_to_booll 8 src []))
 
-(* val bytes_fold_chars: bytes -> ('b -> char -> 'b) -> 'b -> 'b *)
-let bytes_fold_chars_rev str f =
+(* val fold_chars: 'a -> int -> ('a -> int -> char) -> ('b -> char -> 'b) -> 'b -> 'b *)
+let fold_chars_rev buffer size get f =
```

Comment:
I am not sure that this is a good idea to write types in comments since they are not checked by the compiler what about using:
```suggestion
let fold_chars
: 'a -> int -> ('a -> int -> char) -> ('b -> char -> 'b) -> 'b -> 'b =
fun buffer size get f =
```
instead?

Reply:
I want to write a `CONTRIBUTING.md` file at some point to discuss these issues.
I personally would be in favor of doing it systematically, but I also think this needs to be discussed before enforced.

Maybe we can start with a dedicated issue :).

Reply:
Started #20